tutorial/marine/quick_start.md

```diff
@@ -102,48 +102,48 @@ As illustrated in Figure 1 above, the Config.toml file is required for the local
 1. Location of the wasm module to be executed by Marine
 2. One or mode module specific runtime requirements such as module name, logging particulars, local access and memory allocation, Recall, [Wasm memory](https://webassembly.github.io/spec/core/exec/runtime.html#memory-instances) is allocated in linear pages of 64Ki each.
 
-As evident by the existence of the main.rs file, Marine Wasm modules are created as binaries and as we see below, the Marine toolchain compiles our Rust code to the desired wasm32-wasi target. Due to our the Wasm IT target, our Rust files look a little different than Rust application or even wasm-bindgen code.
+As evident by the existence of the main.rs file, Marine Wasm modules are created as binaries and as we see below, the Marine toolchain compiles our Rust code to the desired wasm32-wasi target. Due to the Wasm Interface Types (IT) target, our Rust files look a little different than Rust application or even wasm-bindgen code.
 
 ```rust
 // main.rs
-use fluence::{marine, module_manifest, WasmLoggerBuilder};  <-- 1
+use fluence::{marine, module_manifest, WasmLoggerBuilder};  // <-- 1
 
 module_manifest!();  <-- 2
 
-pub fn main() {
-    WasmLoggerBuilder::new().build().unwrap();  <-- 3
+pub fn main() { // <-- 3
+    WasmLoggerBuilder::new().build().ok();  // <-- 4
 }
 
-#[marine]  <-- 4
-pub fn greeting(name: String) -> String {  <-- 5
+#[marine]  // <-- 5
+pub fn greeting(name: String) -> String {  // <-- 6
     format!("Hi, {}", name)
 }
 
 #[cfg(test)]
 mod tests {
-    use crate::greeting;
-    use fluence_test::marine_test; <-- 6
+    use fluence_test::marine_test; // <-- 7
 
     #[marine_test(config_path = "../Config.toml", modules_dir = "../artifacts")]
     fn test_greeting() {
         let name = "Marine".to_string();
-        let res = greeting(name.clone());
+        let res = greeting.greeting(name.clone());
```

Comment:
I think this deserves an explanation - it's a pretty magical thing. But I'm not sure if it should be explained here or in docs. Overloading reader with info isn't a good idea either. Maybe a comment in the code will do?